test/helper/routeHelper.js

```diff
@@ -7,7 +7,7 @@ const { API_MAJOR_VERSION } = env
 
 export async function apiDocs({ app }) {
   return request(app)
-    .get(`/${API_MAJOR_VERSION}/api-docs`)
+    .get(`/api-docs`)
```

Comment:
this is inconsistent with how we do this elsewhere. For exmaple on matchmaker (https://github.com/digicatapult/sqnc-matchmaker-api/blob/7a32be0c5fd8ab07a8b68201b9307fd402ed7943/src/server.ts) we do 

```js
app.get('/api-docs', (_req, res) => res.json(swaggerJson))
```

Reply:
hmm the motivation was to do the same that matchmaker does like here: https://github.com/digicatapult/sqnc-matchmaker-api/blob/7a32be0c5fd8ab07a8b68201b9307fd402ed7943/src/server.ts#L33